test/test_torch.py

```diff
@@ -6820,10 +6820,24 @@ def inner(self, device, dtype):
     ('size', 'dim', _new_t((1, 2, 3, 4)), lambda t, d: [1], 1e-5, 1e-5, 1e-5, _types, _cpu_types, False),
     ('size', 'neg_dim', _new_t((1, 2, 3, 4)), lambda t, d: [-2], 1e-5, 1e-5, 1e-5, _types, _cpu_types, False),
     ('sort', '', _small_3d_unique, lambda t, d: [], 1e-5, 1e-5, 1e-5, _types, _cpu_types, False),
-    ('sort', 'dim', _small_3d_unique, lambda t, d: [1], 1e-5, 1e-5, 1e-5, _types, _cpu_types, False),
-    ('sort', 'neg_dim', _small_3d_unique, lambda t, d: [-1], 1e-5, 1e-5, 1e-5, _types, _cpu_types, False),
-    ('sort', 'dim_descending', _small_3d_unique, lambda t, d: [1, True], 1e-5, 1e-5, 1e-5, _types, _cpu_types, False),
-    ('sort', 'neg_dim_descending', _small_3d_unique, lambda t, d: [-1, True], 1e-5, 1e-5, 1e-5, _types, _cpu_types, False),
+    ('sort', 'stable', _small_3d_unique, lambda t, d: [0, False, True],
```

Comment:
Instead of updating these tests, implement an OpInfo for sort and remove them. Ping me if you have questions about how to do this.

Reply:
Will do! A small question: Is there a plan for the OpInfo's sample method to support generators? Could be quite handy in implementing several samplers which attach specific use cases and then concatenating them without any additional memory allocation...

Reply:
That's a really cool idea that I hadn't thought about. Would you like to open an issue to suggest/discuss it?